lib/apiservers/engine/backends/container/viccontainer.go

```diff
@@ -30,32 +29,24 @@ type VicContainer struct {
 	HostConfig  *containertypes.HostConfig
 
 	m     sync.RWMutex
-	execs map[string]*types.ExecConfig
+	execs map[string]struct{}
```

Comment:
Probably a dumb question, but where do we keep the `ExecConfig` objects cached now that we aren't storing them here?